collect.py

```diff
@@ -197,6 +235,11 @@ def run_tests(firefox_driver, chrome_driver):
                not os.path.exists('data/%d_chrome.png' % bug['id']):
                 sequence = run_test(bug, 'firefox', firefox_driver)
                 run_test(bug, 'chrome', chrome_driver, sequence)
+
+                with open("data/" + str(bug['id']) + ".txt", 'w') as f:
```

Comment:
Nit: Use `%` instead of `+` as it is more readable

`'data/%d.txt' % bug['id']`